qlib/rl/trainer/vessel.py

```diff
@@ -180,6 +180,7 @@ def train(self, vector_env: FiniteVectorEnv) -> Dict[str, Any]:
             update_result = self.policy.update(sample_size=0, buffer=collector.buffer, **self.update_kwargs)
             res = {**col_result, **update_result}
             self.log_dict(res)
+            del collector
```

Comment:
Doesn't Python delete the reference to the object automatically when the function is over?

Reply:
Please see discussions above.